source/plugins/ruby/out_mdm.rb

```diff
@@ -21,13 +21,18 @@ def initialize
       require_relative "proxy_utils"
 
       @@token_resource_url = "https://monitoring.azure.com/"
+      # AAD auth supported only in public cloud and handle other clouds when enabled
+      # this is unified new token audience for LA AAD MSI auth & metrics
+      @@token_resource_audience = "https://monitor.azure.com/"
```

Comment:
isnt this the same as token_resouce_url? Why do we need another variable?

Reply:
This new token resource audience. See the difference in the resource url name monitor via monitoring.